docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -303,20 +303,20 @@ Individual intervention pages:
      - Value
      - Note
    * - Population size
-     - 100,000
+     - 50,000
      - per random seed/draw combination
    * - Number of draws
-     - 
+     - 10
```

Comment:
is 10 draws, 15 seeds, 50k population required for final results, interim models, or both?